packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -125,37 +108,7 @@ export class PendingStateManager implements IDisposable {
 		private readonly stateHandler: IRuntimeStateHandler,
 		initialLocalState: IPendingLocalState | undefined,
 	) {
-		/**
-		 * Convert old local state format to the new format
-		 * The old format contained "flush" messages as the indicator of batch ends
```

Comment:
Would this cause a back-compat issue when someone is running say `2.0.0-internal.2.3.1` and wants to upgrade to `2.0.0-internal.4.0.0`?

Reply:
I think you're correct. `2.0.0-internal.4.0.0` still needs to be able to read the format with the flush messages since that's what `2.0.0-internal.2.3.0` is returning.